assets/styles/core/_theme.scss

```diff
@@ -0,0 +1,28 @@
+.theme-light {
```

Comment:
I think this theme file could also house some predefined theme colors. Like black backgrround blue background, light gray background.